stable/unifi/templates/ingress.yaml

```diff
@@ -40,4 +40,15 @@ spec:
               {{- end }}
               servicePort: https-gui
   {{- end }}
+  {{- if $unifiedServiceEnabled -}}
+  {{- range .Values.captivePortalService.ingress.hosts }}
+    - host: {{ . }}
+      http:
+        paths:
+          - path: {{ $ingressPath }}
+            backend:
+              serviceName: {{ $fullName }}-captivePortalService
+              servicePort: captive-http
+  {{- end }}
+  {{- end }}
```

Comment:
This block may not be necessary given that there is now a dedicated ingress object for the captive portal.